third_party/requirements.txt

```diff
@@ -4,14 +4,14 @@ clang-format==15.0.7
 cmake-format==0.6.13
 black==24.3.0
 # Tests
-packaging==24.0
+packaging==24.1
 # Generating HTML documentation
-pygments==2.15.1
-sphinxcontrib_applehelp==1.0.4
-sphinxcontrib_devhelp==1.0.2
-sphinxcontrib_htmlhelp==2.0.1
-sphinxcontrib_serializinghtml==1.1.5
-sphinxcontrib_qthelp==1.0.3
+pygments==2.18.0
+sphinxcontrib_applehelp==1.0.8
+sphinxcontrib_devhelp==1.0.6
+sphinxcontrib_htmlhelp==2.0.5
+sphinxcontrib_serializinghtml==1.1.10
+sphinxcontrib_qthelp==1.0.7
 breathe==4.35.0
-sphinx==4.5.0
-sphinx_book_theme==0.3.3
+sphinx==7.3.7
```

Comment:
sphinx version has taken a serious leap - we should make sure the docs layout is still ok with us, I guess

Reply:
Some time ago, I tried bumping up the sphinx versions and it caused a lot of changes in the look

Reply:
This is how you can control updates from PR:
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/managing-pull-requests-for-dependency-updates#managing-dependabot-pull-requests-for-grouped-updates-with-comment-commands